crates/forge-runner/src/running.rs

```diff
@@ -358,3 +378,31 @@ fn get_latest_block_number(url: &Url) -> Result<BlockId> {
         _ => Err(anyhow!("Could not get the latest block number".to_string())),
     }
 }
+
+fn extend_execution_resources(test_execution_syscall_handler: &mut TestExecutionSyscallHandler) {
```

Comment:
I think this file has to be split into separate files eventually, it is growing quite big xD Can we have an issue for that?

Reply:
I think it's not related now since I've moved this function out